ReadingRainbow/FakeData.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  FakeData.swift
+//  ReadingRainbow
+//
+//  Created by Ami Smith on 9/5/23.
+//
+
+import Foundation
+import UIKit
+
+extension Book {
+    struct Fake {
+        static let books = [book1, book2, book3]
+        
+        static let book1Image = UIImage(named: "book1Image")?.pngData() ?? Data()
+        static let book2Image = UIImage(named: "book2Image")?.pngData() ?? Data()
+        static let book3Image = UIImage(named: "book3Image")?.pngData() ?? Data()
+        
+        static let book1Content = [
+            Page(image: Data(), pageNumber: 1, content: ["Page 1 Text"]),
+            Page(image: Data(), pageNumber: 2, content: ["Page 2 Text"])
+        ]
+        
+        static let book2Content = [
+            Page(image: Data(), pageNumber: 1, content: ["Page 1 Text"]),
+            Page(image: Data(), pageNumber: 2, content: ["Page 2 Text"])
+        ]
+        
+        static let book3Content = [
+            Page(image: Data(), pageNumber: 1, content: ["Page 1 Text"]),
+            Page(image: Data(), pageNumber: 2, content: ["Page 2 Text"])
+        ]
+        
+        static let book1 = Book(image: book1Image, title: "Harry Potter and the Sorcerer's Stone", pages: book1Content, uniqueWords: ["magic", "wizard", "spell"], paragraphs: ["Paragraph 1 Text", "Paragraph 2 Text"])
```

Comment:
In an ideal world the fake data would be as close to real as possible (where unique words and paragraphs match the image)